src/app/containers/Header/index.jsx

```diff
@@ -3,7 +3,7 @@ import BrandContainer from '../Brand';
 
 const HeaderContainer = () => (
   <header role="banner">
-    <BrandContainer />
+    <BrandContainer isHeader />
```

Comment:
I'm a bit late with this one but wouldn't it be better to just do `<BrandContainer borderTop />` here and `<BrandContainer borderBottom />` in the footer and then in `Brand` you can spread the props on the component like

```jsx
    <Brand
      product={product}
      serviceLocalisedName={serviceLocalizedName}
      svgHeight={svgMaxHeight}
      minWidth={minWidth}
      maxWidth={maxWidth}
      svg={brandSVG}
      url={`/${service}`}
      {...props}
    />
```

Reply:
@jroebu14 Not too late. Made that change  https://github.com/bbc/simorgh/pull/2146